README.md

```diff
@@ -190,6 +190,9 @@ The lack of any of such annotations means `strict` mode. If both are set, `clust
 > NOTE: next release will consolidate this into a single `sealedsecrets.bitnami.com/scope` annotation.
 > NOTE: next release will enhance the kubeseal tooling so you don't have to fiddle with these annotations.
 
+By default, the `strict` scope is used, which means *both the namespace and name are part of the
```

Comment:
This is described in the paragraph at line 173.
But I know that people faced with a wall of text tend to not read it, so it's ok if we repeat, summarize and highlight common pitfalls.

That said, shouldn't we add this text above at line 179? I just noticed that I butchered that list (see line 181 that only says "you"):

What about:

```diff
- You can select the "scope":
+ Depending on the scope, the name and/or namespace become *part of the encrypted data* and thus changing name and/or namespace would lead to "decryption error". These are the possible scopes:

- * strict (default)
+ * strict (default): the secret must be sealed with exactly the same *name* and *namespace*
  * namespace-wide: you can freely *rename* the sealed secret within a given namespace
- * cluster-wide: you
+ * cluster-wide: the secret can be unsealed in *any* namespace and can be given *any* name.
```

Reply:
I've put the scope explanation into (only) this bullet point list, and added a distinction to secret _data keys_ which can be freely renamed.